### ISSUE TITLE
Crash while processing Doxygen files from wxWidgets

### ISSUE THREAD

**Issue by tritao** [matusnovak/doxybook2#3]
Just tried a fresh build with XML files from wxWidgets build and it crashes.

[xml.zip](https://github.com/matusnovak/doxybook2/files/4343642/xml.zip)

### Command:
`./build/src/DoxybookCli/doxybook2 --input out/xml --output out/json --json`

### Stack trace:

```
* thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BAD_ACCESS (code=EXC_I386_GPFLT)
  * frame #0: 0x000000010015b499 doxybook2`std::__1::vector<Doxybook2::Node::ClassReference, std::__1::allocator<Doxybook2::Node::ClassReference> >::vector(std::__1::vector<Doxybook2::Node::ClassReference, std::__1::allocator<Doxybook2::Node::ClassReference> > const&) [inlined] std::__1::vector<Doxybook2::Node::ClassReference, std::__1::allocator<Doxybook2::Node::ClassReference> >::size(this=0x00080000000000d0 size=0) const at vector:633
    frame #1: 0x000000010015b499 doxybook2`std::__1::vector<Doxybook2::Node::ClassReference, std::__1::allocator<Doxybook2::Node::ClassReference> >::vector(this=0x00007ffeefbf9cc8 size=0, __x=size=0) at vector:1209
    frame #2: 0x0000000100140a3d doxybook2`std::__1::vector<Doxybook2::Node::ClassReference, std::__1::allocator<Doxybook2::Node::ClassReference> >::vector(this=0x00007ffeefbf9cc8 size=0, __x=size=0) at vector:1205
    frame #3: 0x0000000100122147 doxybook2`Doxybook2::Node::getAllBaseClasses(this=0x0008000000000000, cache=size=25280) at Node.cpp:677
    frame #4: 0x000000010012257b doxybook2`Doxybook2::Node::getAllBaseClasses(this=0x0000000101d0ce28, cache=size=25280) at Node.cpp:686
    frame #5: 0x000000010012257b doxybook2`Doxybook2::Node::getAllBaseClasses(this=0x0000000101d0b688, cache=size=25280) at Node.cpp:686
    frame #6: 0x000000010011f043 doxybook2`Doxybook2::Node::finalize(this=0x0000000101d0b688, config=0x00007ffeefbfc308, plainPrinter=0x00007ffeefbfc000, markdownPrinter=0x00007ffeefbfc048, cache=size=25280) at Node.cpp:369
    frame #7: 0x00000001000b6a6f doxybook2`Doxybook2::Doxygen::finalizeRecursively(this=0x00007ffeefbfc078, plainPrinter=0x00007ffeefbfc000, markdownPrinter=0x00007ffeefbfc048, node=std::__1::shared_ptr<Doxybook2::Node>::element_type @ 0x00000001007022e8 strong=1 weak=1) at Doxygen.cpp:194
    frame #8: 0x00000001000b682a doxybook2`Doxybook2::Doxygen::finalize(this=0x00007ffeefbfc078, plainPrinter=0x00007ffeefbfc000, markdownPrinter=0x00007ffeefbfc048) at Doxygen.cpp:186
    frame #9: 0x00000001000045fb doxybook2`main(argc=6, argv=0x00007ffeefbff688) at main.cpp:132
    frame #10: 0x00007fff6274d115 libdyld.dylib`start + 1
    frame #11: 0x00007fff6274d115 libdyld.dylib`start + 1
```

**Comment by matusnovak** [comment]
Hi @tritao 

Thanks for the xml.zip and the stacktrace, it helped a lot.

I have managed to fix the problem, the fix is in the [releases/tag/v1.0.3](https://github.com/matusnovak/doxybook2/releases/tag/v1.0.3).

**Comment by tritao** [comment]
Sorry forgot to close this, thanks a lot for the fast fix. 👍
